src/embed_builder.ts

```diff
@@ -210,18 +211,33 @@ export class EmbedBuilder<T> {
   /**
    * Allows auth url to be specified
    *
-   * @param authUrl
+   * @param authUrl URL to endpoint that can sign Looker SSO URLs
    */
 
   withAuthUrl (authUrl: string) {
-    if (!this._hostSettings.authUrl) {
-      this._hostSettings.authUrl = authUrl
-    } else if (this._hostSettings.authUrl !== authUrl) {
+    if (!this._hostSettings.auth?.url) {
+      this._hostSettings.auth = { url: authUrl }
+    } else if (this._hostSettings.auth.url !== authUrl) {
       throw new Error('not allowed to change auth url')
     }
     return this
   }
 
+  /**
+   * Allows auth url to be specified
```

Comment:
Technically this is more than just the auth URL. Maybe a little more explanatory blurb?